netkan/netkan/scheduler.py

```diff
@@ -91,8 +95,17 @@ def volume_credits_percent(self, cloudwatch: 'boto3.CloudWatch.Client', instance
             logging.error("Couldn't acquire Volume Credit Stats")
         return int(creds)
 
-    def can_schedule(self, max_queued: int, dev: bool = False, min_cpu: int = 50, min_io: int = 70) -> bool:
+    def can_schedule(self, max_queued: int, dev: bool = False,
+                     min_cpu: int = 50, min_io: int = 70, min_gh: int = 1500) -> bool:
         if not dev:
+            github_remaining = github_limit_remaining(self.github_token)
+            if github_remaining < min_gh:
```

Comment:
It would be nice if we could turn this check off for the SpaceDock webhooks-only pass, since it doesn't use the GitHub API. Maybe make `min_gh` a command line parameter?

Reply:
That's a good idea, might leave for a future iteration. It shouldn't be a problem for now. It could use a refactor, we're stuffing a lot of logic and kwargs into that function!